playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,99 @@
+[all:vars]
+app_dns_prefix=apps
+public_hosted_zone=example.com
+load_balancer_hostname=lb0.{{public_hosted_zone}}
+openshift_master_cluster_hostname="{{ load_balancer_hostname }}"
+openshift_master_cluster_public_hostname=openshift-master.{{ public_hosted_zone }}
+openshift_master_default_subdomain="{{ app_dns_prefix }}.{{ public_hosted_zone }}"
+openshift_public_hostname="{{openshift_master_cluster_public_hostname}}"
+
+[OSEv3:children]
+nodes
+masters
+etcd
+lb
+
+[OSEv3:vars]
+# General variables
+ansible_ssh_user=root
+console_port=8443
+debug_level=2
+deployment_type=openshift-enterprise
+openshift_debug_level="{{ debug_level }}"
+openshift_deployment_type="{{ deployment_type }}"
+openshift_master_cluster_method=native
+openshift_node_debug_level="{{ node_debug_level | default(debug_level, true) }}"
+openshift_release='3.10'
+openshift_vers=v3_10
+
+# RHV Specific Settings
+openshift_enable_service_catalog=False
+# openshift_cloudprovider_kind=ovirt
+# openshift_cloudprovider_ovirt_auth_url=
+# openshift_cloudprovider_ovirt_username=
+# openshift_cloudprovider_ovirt_password=
+
+# Pod Networking
+os_sdn_network_plugin_name=redhat/openshift-ovs-networkpolicy
+
+# Registry
+openshift_hosted_registry_replicas=1
+openshift_hosted_registry_storage_kind=nfs
+openshift_hosted_registry_storage_access_modes=['ReadWriteMany']
+openshift_hosted_registry_storage_host=
+openshift_hosted_registry_storage_nfs_directory=/var/lib/exports
+openshift_hosted_registry_storage_volume_name=registryvol
+openshift_hosted_registry_storage_volume_size=20Gi
+openshift_enable_unsupported_configurations=True
+
+# Authentication
+openshift_master_identity_providers="[{'name': 'htpasswd_auth', 'login': 'True', 'challenge': 'True', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]"
```

Comment:
In my inventory file it is set to:
openshift_master_identity_providers=[{'mappingMethod': 'lookup', 'challenge': 'true', 'login': 'true', 'kind': 'AllowAllPasswordIdentityProvider', 'name': 'allow_all'}]

Can you please help me figure out the difference?

Reply:
`HTPasswdPasswordIdentityProvider` sets up an htpasswd file on all three masters, and you have to add an entry to that, my example being myuser with password changeme.

`AllowAllPasswordIdentityProvider` allows in any user with any userid/password combination.

Neither have anything specific to do with RHV, so I'd rather remove the section and leave it to the user to specify their own.

Reply:
Is there a default for openshift_master_identity_providers?  If not, I think we should give something and I'll add automation for it.